tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -263,6 +266,8 @@ sub post_fail_hook {
     $self->post_run_test;
     save_screenshot;
 
+    check_host_health;
+
```

Comment:
I think one ```check_host_health``` is enough in post_fail_hook.

Reply:
It is only called once.

Reply:
If test fails, it is called twice ?

Reply:
why twice? if post_fail_hook() called once, it will be called once. Except some tests with `fatal => 0`, such as sriov test, it may called again in other tests' post_fail_hook if they fail again.

Or do you mean it only be called in either of post_fail_hook or prepare_test? it need not be called in both places.

Reply:
There are two
```
check_host_health
```
in this virt_autotest_base.pm module. I think one in post_fail_hook is enough.

Reply:
if it was not called before test run, 

- how do you know it the host is ok before your test?
- if test passes, nobody would be awared of there were calltrace/coredump .